### ISSUE TITLE
Dashboard graphs don't always resize like they should

### ISSUE THREAD

**Issue by laurijamsa** [ruuvi/com.ruuvi.station.webui#241]
<img width="618" alt="image" src="https://user-images.githubusercontent.com/1311912/197466039-8cde47cf-2d86-4f4e-a525-021555b0a18c.png">

<img width="378" alt="image" src="https://user-images.githubusercontent.com/1311912/197466085-057547df-e8c5-48dd-9dcc-1c054020d688.png">

**Comment by laurijamsa** [comment]
I still see this sometimes on the dev:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/1311912/199464016-cae52027-dfad-460f-b8f4-4cfd2fc96f64.png">

**Comment by laurijamsa** [comment]
looks good on the dev. cannot reproduce the issue

**Comment by laurijamsa** [comment]
Found some issues again when zooming on Chrome browser (90% zoom level breaks the graphs). Another thing I noticed is that for example when I use 150% zoom level, I see 3 sensor columns but when I zoom out to 110% and 90%, only 2 columns are shown. Not sure what's reasoning behind this exactly. Video:

https://user-images.githubusercontent.com/1311912/202945056-03677412-6029-471c-8cf7-1d0b93aeae3a.mov
